Design & modified build settings

PolinaShneider opened the pull request:
Added postcss-loader

PolinaShneider committed:
Design & modified build settings

Added postcss-loader

PolinaShneider commented:
![screencapture-capella-local-1509057152159](https://user-images.githubusercontent.com/15448200/32080171-defa8e4c-bab6-11e7-9554-a95039242ede.png)

talyguryn commented on public/app/img/instagram-logo.svg:
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
надо прогнать через оптимайзер https://jakearchibald.github.io/svgomg/

talyguryn commented on public/app/css/pages/index.css:
```diff
@@ -0,0 +1,100 @@
+.capella-background {
```
это не бэм 😒

talyguryn commented on public/app/css/pages/index.css:
```diff
@@ -0,0 +1,100 @@
+    letter-spacing: 5px;
+    font-variant: small-caps;
+}
+.capella-btn {
```
btn? не жадничай. добавь буквы

talyguryn commented on public/app/css/pages/index.css:
```diff
@@ -0,0 +1,100 @@
+    vertical-align: middle;
+    cursor: pointer;
+}
+.capella-btn:hover {
```
у тебя подключен постцсс

```
.upload-button {
    ...

    &:hover {
        ...
    }

    &:before {
        ...
    }
}
```

PolinaShneider committed:
Renamed according to BEM

Also named selectors using post-css,
Optimized svg

PolinaShneider committed:
Connected HTML to functional

PolinaShneider committed:
Conflicts

PolinaShneider committed:
Merge branch 'master' into design

PolinaShneider committed:
Change element names back

PolinaShneider committed:
Pixel-perfect

At least, close to :)

PolinaShneider commented:
![screencapture-capella-local-1509399090991](https://user-images.githubusercontent.com/15448200/32196810-f3b967bc-bdd2-11e7-903f-9280da74d6e7.png)

PolinaShneider committed:
Even more similar to design

* Changed border to svg, with more spaced dashes
* Previous logo white circle was uneven, made it through css :(

Here is the link, updated: https://polinashneider.github.io/Sandbox/capella.html

PolinaShneider committed:
Merge branch 'master' into design

PolinaShneider committed:
Update from master

Some design fixes

gohabereg commented on public/app/css/pages/index.css:
```diff
@@ -0,0 +1,144 @@
+    background-size: cover;
+    text-align: center;
+    position: fixed;
+    top: 0;
```
Перенеси эти свойства в начало

gohabereg commented on public/app/css/pages/index.css:
```diff
@@ -0,0 +1,144 @@
+    box-sizing: border-box;
+    background: url("/public/app/img/Dashed-border.svg") no-repeat;
+    background-size: 100% 100%;
+    margin: 0 auto;
```
И тут тоже неплохо бы отсортировать свойства

gohabereg commented on public/app/css/pages/index.css:
```diff
@@ -0,0 +1,144 @@
+    &__button {
+        display: inline-block;
+        background: #6c0bd1;
+        margin-top: 35px;
```
И тут

gohabereg commented on public/app/css/pages/index.css:
```diff
@@ -0,0 +1,144 @@
+    &__input {
+        border-color: transparent;
+        background: transparent;
+        -webkit-appearance: none;
```
Это точно будет работать во всех браузерах?

PolinaShneider committed:
Optimized svg

Created folder for svg
Deleted unused svg
CSS properties order refactoring

PolinaShneider committed:
Merge branch 'master' of https://github.com/codex-team/capella into design

Update from master

PolinaShneider committed:
Update from master

talyguryn committed:
Merge branch 'master' into design

# Conflicts:
#	.gitignore

PolinaShneider committed:
Border fix

PolinaShneider committed:
Merge branch 'design' of https://github.com/codex-team/capella into design

PolinaShneider committed:
Fix height, remove mobile border

PolinaShneider committed:
Self-review :)

Optimized JPEG image
Got rid of absolute positioning
Got rid of unused SVG files
Refactored CSS properties order

If not possible to test on alpha right now, see updated version here: https://polinashneider.github.io/Sandbox/capella.html

PolinaShneider committed:
Lots of fixes

Got rid of fixed positioning
Changed logo
Fixed drag'n'drop frame size
Added attribute lang
Changed background
Renamed classes
Changed font

PolinaShneider committed:
Background, finally

PolinaShneider committed:
Bg optimize

PolinaShneider committed:
Font, bg, remove extra class

PolinaShneider committed:
Bg color

PolinaShneider committed:
Change bg, fix button

PolinaShneider committed:
Change color and button padding

neSpecc commented on public/app/css/pages/index.css:
```diff
@@ -0,0 +1,116 @@
+        flex-grow: 1;
+        align-items: center;
+        justify-content: center;
+        width: 100%;
```
лишнее

neSpecc commented on public/app/css/pages/index.css:
```diff
@@ -0,0 +1,116 @@
+        width: 100%;
+        box-sizing: border-box;
+        margin: 30px;
+        padding: 80px 0;
```
давай __content не через паддинг, а как-то по другому центрируем, чтобы на больших экранах он тоже был в центре по вертикали

neSpecc commented on public/app/css/pages/index.css:
```diff
@@ -0,0 +1,116 @@
+        margin-top: 32px;
+        border: 0;
+        border-radius: 25px;
+        background: #6c0bd1;
```
вынеси цвет в переменную

neSpecc commented on src/view/index.php:
```diff
@@ -1,6 +1,26 @@
+    <head>
+        <title>Main</title>
+        <link rel="icon" href="./favicon.ico" type="image/x-icon">
+        <link rel="shortcut icon" href="./favicon.ico" type="image/x-icon">
```
пути странные. И что за иконка?

neSpecc replied:
и type странный, это что?

PolinaShneider committed:
Variables

PolinaShneider committed:
Placeholder
